tests/console/yast2_cmdline.pm

```diff
@@ -44,7 +44,7 @@ sub run_yast_cli_test {
 
 sub run {
     select_console 'root-console';
-
+    return if (systemctl("status wicked.service", ignore_failure => 1) != 0);
```

Comment:
Hmm, I believe that we need at least record info here, otherwise it's not even visible that we did nothing and have passing test. Next step will be to get proper setup to make this test suite work in TW.

Reply:
I will  create another as this has been merged, and i will ask you to review it

Reply:
Perfect! Thanks a lot!

Reply:
So i ended up with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9862 and followup ticket have been created to solve the problem in o3